tests/common/Configuration.cs

```diff
@@ -43,6 +43,15 @@ static partial class Configuration
 		public static bool include_watchos;
 		public static bool include_device;
 
+		static Version xcode_version;
+		public static Version XcodeVersion {
+			get {
+				if (xcode_version == null)
+					xcode_version = Version.Parse (XcodeVersionString);
```

Comment:
`TryParse` and log failures ?

Reply:
If we fail to parse the Xcode version things are going sideways pretty quickly anyway.

Reply:
true, but will it be easy to diagnose for a customer reporting the issue ?

Reply:
This is in tests/, so it's not customer-facing.